### PR TITLE
Community Roundup #25 blog post

### PR DESCRIPTION
**Pull request by matthewjohnston4**

### CODE REVIEW

**Comment by zertosh** [comment]
hey @matthewjohnston4 any chance we can get a [babel-sublime](https://github.com/babel/babel-sublime) mentioned? we have awesome ES6 + JSX support plus other [goodies](https://github.com/babel/babel-sublime#screenshots)

**Comment by vjeux** [comment]
@matthewjohnston4 can you include babel-sublime and ping me when it's done so that I can merge it in?

**Comment by zpao** [comment]
2 things.
1. Can we get a mention of [react-meteor](https://github.com/reactjs/react-meteor) in here? It was updated recently. [React.NET](https://github.com/reactjs/React.NET) also had a release yesterday.
2. Can we hold off on publishing this until tomorrow? I have an RC2 blog post I want to get up today.

**Comment by vjeux** [comment]
@zpao maybe you can mention React.net as part of the RC2 announcement

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Interest in React has been exploding recently, so it's a good time to explore some great recent tutorials and videos covering getting started. 
+
+Ryan Clarke ([rynclark](https://github.com/rynclark)) provides a [great overview of the basics of React](http://ryanclark.me/getting-started-with-react/) with the goal of building a really simple dropdown nav. 
+
+Formidable Labs ([FormidableLabs](https://github.com/FormidableLabs)) and [Seattle JS](http://www.meetup.com/seattlejs/) recently hosted a series of React, Flux, and Flow workshops, and the first part is available to watch online:
+
+<iframe width="560" height="315" src="//www.youtube.com/embed/[Pd6Ub7Ju2RM](https://www.youtube.com/watch?v=Pd6Ub7Ju2RM)" frameborder="0" allowfullscreen></iframe>
+
+AEFlash ([aearly](https://github.com/aearly)) writeup [some best practices and tips](http://aeflash.com/2015-02/react-tips-and-best-practices.html) to help you avoid potential pitfalls when developing with React.
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: This isn't a sentence.

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Ryan Clarke ([rynclark](https://github.com/rynclark)) provides a [great overview of the basics of React](http://ryanclark.me/getting-started-with-react/) with the goal of building a really simple dropdown nav. 
+
+Formidable Labs ([FormidableLabs](https://github.com/FormidableLabs)) and [Seattle JS](http://www.meetup.com/seattlejs/) recently hosted a series of React, Flux, and Flow workshops, and the first part is available to watch online:
+
+<iframe width="560" height="315" src="//www.youtube.com/embed/[Pd6Ub7Ju2RM](https://www.youtube.com/watch?v=Pd6Ub7Ju2RM)" frameborder="0" allowfullscreen></iframe>
+
+AEFlash ([aearly](https://github.com/aearly)) writeup [some best practices and tips](http://aeflash.com/2015-02/react-tips-and-best-practices.html) to help you avoid potential pitfalls when developing with React. 
+
+Black Mutt Media [take us through their usage of React](http://blackmuttmedia.com/blog/react-tmdb-api/) and Ruby to build an autocomplete field, and some of the pitfalls they encountered along the way.
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: "takes"

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Our own [sebmarkbage ](https://github.com/sebmarkbage)was on the[ Web Platform Podcast](http://thewebplatform.libsyn.com/31-building-with-reactjs) to have a chat about all aspects of React.
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: Sebastian Markbåge

Also, put the spaces outside the links (you do that twice).

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Community Additions
+
+Formidable Labs ([FormidableLabs](https://github.com/FormidableLabs)) have been busy, as they've also[ just launched Radium](http://projects.formidablelabs.com/radium/), a React component that provides you with the ability to use inline styles instead of CSS. They're also [looking for some help](http://projects.formidablelabs.com/radium-bootstrap/) contributing to a Radium Bootstrap implementation.
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: Same note about moving the space outside the link.

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Our own [sebmarkbage ](https://github.com/sebmarkbage)was on the[ Web Platform Podcast](http://thewebplatform.libsyn.com/31-building-with-reactjs) to have a chat about all aspects of React.
+
+## Community Additions
+
+Formidable Labs ([FormidableLabs](https://github.com/FormidableLabs)) have been busy, as they've also[ just launched Radium](http://projects.formidablelabs.com/radium/), a React component that provides you with the ability to use inline styles instead of CSS. They're also [looking for some help](http://projects.formidablelabs.com/radium-bootstrap/) contributing to a Radium Bootstrap implementation. 
+
+[Reactiflux.com](http://reactiflux.com/) is a new Slack community based around (you guessed it!) React, and Flux. 
+
+[React Week](http://reactweek.com/) is a recently launched, week-long learning workshop for React, Flux, and other related technologies, run by Ryan Florence ([ryanflorence](https://github.com/ryanflorence)).
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: Might be worth mentioning that this is *next week*

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Rebuilding with React
+
+Rich Manalang ([rmanalan](https://github.com/rmanalan)) from Atlassian [explains why](https://developer.atlassian.com/blog/2015/02/rebuilding-hipchat-with-react/) they rebuilt their HipChat web client from scratch using React, and how they're already using it to rebuild their native desktop clients. 
+
+[Andrew Hillel](https://twitter.com/andyhillel) of the BBC gives [an excellent and thorough breakdown](http://www.bbc.co.uk/blogs/internet/entries/47a96d23-ae04-444e-808f-678e6809765d) of the stack they used to rebuild their homepage, with React as an integral part of the front-end. 
+
+A team from New Zealand called [Atomic ](https://atomic.io/)is[ building web and mobile prototyping and design tools ](http://thenextweb.com/creativity/2015/02/19/meet-atomic-missing-tool-interface-design-thats-entirely-browser/)entirely in-browser, and as co-founder [Darryl Gray](http://twitter.com/darrylgray) says, “React.js “totally changed” the fact that browser performance often wasn’t good enough for complex tools like this.”.
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: spaces

File: docs/_posts/2015-03-03-community-roundup-25.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+Polarr ([polarrco](https://github.com/Polarrco)) have rebuilt [their browser-based photo editor ](http://polarrist.tumblr.com/post/111290422225/polarr-photo-editor-2-0-alpha-is-here)with React.
```
Comment (zpao) [inline docs/_posts/2015-03-03-community-roundup-25.md]: spaces

**Comment by zertosh** [comment]
thanks @vjeux and @matthewjohnston4

**Pull request merged**